### ISSUE TITLE
Dataset split error?

### ISSUE THREAD

**Issue by hszhoushen** [KaihuaTang/Scene-Graph-Benchmark.pytorch#152]
## ❓ Questions and Help
One issue I was not sure about is that the dataset of VG in total has about 108, 000 images, where 70% are used as the training set (5000 are selected as Val set from the training set), and 30% are used as the test set. However, I test the code and found that in the repository there are about 57723 as training set and 5000 as Val set. However, according to the real dataset size, the actual training set should be more than 108,000*0.7=75600-5000~=70600 images, so what's wrong with the dataset split in the program?

**Comment by hszhoushen** [comment]
Training set
============
57723
6 images distributed at each GPU
============

Test set
============
5000
1 image distributed at each GPU
============
